src/data/clean_dataset.py

```diff
@@ -1,6 +1,13 @@
 import pandas as pd
 import os, sys
-
+import numpy as np
+    from sklearn.datasets.samples_generator import (make_blobs,
+                                                    make_circles,
+                                                    make_moons)
+    from sklearn.cluster import KMeans, SpectralClustering
+    from sklearn.preprocessing import StandardScaler
+    from sklearn.metrics import silhouette_samples, silhouette_score
+    from warnings import filterwarnings
```

Comment:
Take care with indentation!
```suggestion
from sklearn.datasets.samples_generator import (make_blobs,
                                                make_circles,
                                                make_moons)
from sklearn.cluster import KMeans, SpectralClustering
from sklearn.preprocessing import StandardScaler
from sklearn.metrics import silhouette_samples, silhouette_score
from warnings import filterwarnings
```